### PR TITLE
Various style changes

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What's this PR do?
- Changes the button color to be more inline with the MIT red.
- Tightens up the spacing in the learner search, the user chip hover.
- Adds a hover state for the learner search rows
- changes the color of the grade histogram
- changes the color of the "Clear Filters" background
- Changes the color of the "Email Learners" button and add a hover state
- Changes the background color of the sort by dropdown
- Changes the color of the financial aid box

### How to Test

Especially see if there are any buttons in the old color, or hover state color problems.

### CODE REVIEW

**Comment by pdpinch** [comment]
Rob, can you break this down into smaller PRs? 8 changes spread across 12 files are going to slow the review down. 

If you need some help with the git part, I can do it or you can grab someone else.

**Comment by aliceriot** [comment]
Looking this over I think it's manageable as a single PR

File: static/scss/user-chip.scss [hunk 1]
```diff
@@ -70,3 +71,8 @@
     display: block;
   }
 }
+
+.learner-result:hover {
+  background: $bg-light-gray;
+}
```
Comment (aliceriot) [inline static/scss/user-chip.scss]: could you move this into `search-page.scss`, where the rest of the styling for `.learner-result` is?

File: static/scss/navbar.scss [hunk 2]
```diff
@@ -285,3 +282,7 @@ $header_font_color: white;
   text-decoration: none;
   font-weight: 300;
 }
+
+.mdl-layout-title > a:hover, .mdl-layout-title > a:focus {
+  color: white;
+}
```
Comment (aliceriot) [inline static/scss/navbar.scss]: what's this change for? just curious.

Comment (roberthouse54) [inline static/scss/navbar.scss]: This keeps the "MITx Micromasters" link in the navbar from changing to the default hover color.

**Pull request merged**